### PR TITLE
Removed scraping, added bot account checking

### PR DESCRIPTION
**Pull request by DinosParkour**
Since bots now use OAuth2, scraping isn't an issue. (or at least it shouldn't be one?)
Also added a recommendation about checking if a user is a bot account before replying to the command.

### CODE REVIEW

**Comment by meew0** [comment]
Seems good to me, I'm going to wait until at least May 2 to join it when non bot accounts are finally discouraged. Any other opinions?

**Comment by geo1088** [comment]
Definitely agree that invite scraping is obsolete now that bot accounts are fully implemented. However, I'm not entirely sure what's meant by the last point about adding zero-width spaces; @DinosParkour could you clarify that a bit?

**Comment by DinosParkour** [comment]
@geo1088 You can start your messages with a ZWSP so that no other bots can be triggered (in case they don't check for the author being a bot account)

**Comment by geo1088** [comment]
Ooh, I really like that idea, nice.

**Comment by geo1088** [comment]
👍 Looks all good to me now.

File: README.md [hunk 1]
```diff
@@ -13,28 +13,26 @@ Should you opt to use a prefix for your bot, consider using words (`owl`) or
...
+7. **Be respectful of Discord's API**. Bots that abuse and misuse the Discord
 API ruin things for everyone. Make sure to factor in rate-limiting and backoff
 in your bot code, and be intelligent about using the API. Make sure to ask for
 help if you're unsure about the right way to implement things.
-9. **Ignore your own messages**. This helps prevent infinite self-loops and
-potential security exploits.
+8. **Ignore both your own and other bots' messages**. This helps prevent infinite
+self-loops and potential security exploits. Using a zero width space such as `\u200B`
+and `\u180E` in the beginning of each message also prevents your bot from
+trigerring other bots' commands.
```
Comment (meew0) [inline README.md]: There's a spelling mistake here, it should be "triggering"

Comment (DinosParkour) [inline README.md]: my bad :astonished:

**Pull request merged**